bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/PerChannelBookieClient.java

```diff
@@ -845,6 +846,56 @@ private void writeAndFlush(final Channel channel,
             errorOut(key);
         }
     }
+    
+    void errorOutReadKey(final CompletionKey key) {
```

Comment:
This code isn't valid anymore. Erroring out is now part of the completionValue, so this should be handled there.